core/src/com/unciv/models/ruleset/tile/TileResource.kt

```diff
@@ -37,8 +37,10 @@ class TileResource : RulesetStatsObject(), GameResource {
     var majorDepositAmount: DepositAmount = DepositAmount()
     var minorDepositAmount: DepositAmount = DepositAmount()
     
+    @delegate:Transient
```

Comment:
We shouldn't be serializing this in any case, doesn't need this
If it DOES that means we're serializing, which we should definitely get rid of